src/main/webapp/style.css

```diff
@@ -60,35 +60,20 @@ h1 {
   box-shadow: 0 2px 6px rgba(0, 0, 0, 0.4);
   background-color: #fff;
   pointer-events: all;
-  overflow: hidden;
+  overflow-x: hidden;
+  overflow-y: auto;
```

Comment:
We should maintain the overflow to hidden. Instead change the max height for the pac card so that it can accommodate your changes. It would be best to avoid scrollbars especially in mobile as it isn't as obvious as it seems on the pc page.

Reply:
My concern is that later on when we add more filters the pac card will start to completely fill up the screen on the mobile phone - shouldn't we be avoiding it?

Reply:
I agree, no worries, we can keep it then. I'll work on making the filter into a selectable list so that we don't take up so much space on that